DEVELOPING.MD

```diff
@@ -144,6 +144,10 @@ vCenter:
   password: <password>
 ```
 
+If you want to test that CPI is functioning properly, add the `node.cloudprovider.kubernetes.io/uninitialized=true:NoSchedule` taint to the cluster nodes. When CPI is installed, the kubelet should remove the taints.
```

Comment:
not sure how detailed you want to be here, but I would specify that the taint is added before installing the chart. i.e. 
```
If you want to test that CPI is functioning properly, add the `node.cloudprovider.kubernetes.io/uninitialized=true:NoSchedule` taint to the cluster nodes. Then, install CPI and observe that kubelet has removed the taints.
```